pkgs/by-name/eq/equ/package.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitLab,
+  makeBinaryWrapper,
+  zlib,
+  xorg,
+  fontconfig,
+}:
+
+buildGoModule rec {
+  pname = "equ";
+  version = "1.2.5";
+
+  src = fetchFromGitLab {
+    owner = "cznic";
+    repo = "equ";
+    rev = "v${version}";
+    hash = "sha256-+BCsJhXJgJc++9W4kf4xI6XwSV91R+jxqPOoRH9bd48=";
+  };
+
+  vendorHash = "sha256-I2yLiT1vYvv7x/lfsWp/5A/8800DNSKx8fRk6XeMmnk=";
+
+  # From some reason, wrapping is needed for the GUI - the Go package doesn't
+  # seem to demand these packages automatically.
+  nativeBuildInputs = [
+    makeBinaryWrapper
+  ];
+  postInstall = ''
+    wrapProgram "$out/bin/equ" --prefix LD_LIBRARY_PATH ":" ${
```

Comment:
If #418244 gets merged, you will be able to use `LD_AUDIT` instead of `LD_LIBRARY_PATH`.

@doronbehar Could you help reviewing #418244, please?

Reply:
In order to use `LD_AUDIT`, you will need to build a DSO to handle audit events, and `ld-audit-search-mod` provides such a DSO.